es/about/documentación-para-traductores.md

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Should we used an English name and add a blank page or a translated version on `en/about/` directory